src/nrnoc/extcelln.c

```diff
@@ -6,13 +6,19 @@
 #include	"section.h"
 #include	"nrniv_mf.h"
 #include	"hocassrt.h"
+#include	"parse.h"
 
 extern int cvode_active_;
 extern int nrn_use_daspk_;
+extern void nrn_delete_prop_pool(int type);
 
 #if EXTRACELLULAR
 
+int nrn_nlayer_extracellular = EXTRACELLULAR;
```

Comment:
nitpick : It would be clear to rename this macro to `N_EXTRACELLULAR_LAYERS` or something similar.

Also, the same macro is used with `#if`. Does this mean one can also set it to `0` and code will compile?

Reply:
It is supposed to be able to compile with ```#define EXTRACELLULAR 0``` as all relevant fragments are supposed to be encapsulated in ```#if EXTRACELLULAR```.  I don't know that it has ever been used with a value of 0. There are 60 such fragments only one of which depends on the number of layers (1 or >1) which I need to  modify to depend on nrn_nlayer_extracellular. As this define was used in the past to define nlayer and  I wanted to minimally change the existing code, I prefer to keep the existing define and interpret its value as the default value for nrn_nlayer_extracellular.

Reply:
Ok understood!